File: Curso de Java/Exer/src/Outros/lista64Exer/Exer25.java

/*******************************************************************************
* 25) Tendo como dados de entrada a altura e o sexo de uma pessoa, faça um
* programa que calcule seu peso ideal, utilizando as seguintes fórmulas:
* (h = altura)
*   - Para homens: (72.7*h) - 58
*   - Para mulheres: (62.1 *h) - 44.7
*******************************************************************************/

import java.util.Locale;
import java.util.Scanner;

public class Exer25 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		System.out.print("Informe sua altura, em metros: ");
		double height = sc.nextDouble();

		System.out.print("Informe seu sexo: (M - F) ");
		char sex = sc.next().toLowerCase().charAt(0);

		double man = (72.7 * height) - 58;
		double woman = (62.1 * height) - 44.7;

		if (sex == 'm') {
			System.out.println("Seu peso ideal é: " + man);
		} else {
			System.out.println("Seu peso ideal é: " + woman);
		}

		sc.close();
	}

}

File: CursoLuizOtavio/CursoPython3/exer/lista_64_Exer/exer25.py
"""
25) Tendo como dados de entrada a altura e o sexo de uma pessoa, faça um programa que calcule seu peso ideal, 
utilizando as seguintes fórmulas: (h = altura) 
- Para homens: (72.7*h) - 58 
- Para mulheres: (62.1 *h) - 44.7 
"""

height = float(input("Digite sua altura: "))
sex = input("Digite seu sexo [M/F]: ").upper()

if (sex == "M"):
    weight = (72.7 * height) - 58
    print(f"Seu peso ideal é: {weight}")
elif (sex == "F"):
    weight = (62.1 * height) - 44.7
    print(f"Seu peso ideal é: {weight}")
else:
    print("Sexo inválido.")
